test/test_main.py

```diff
@@ -696,36 +749,42 @@ def test_excepthook(self):
             '    raise RuntimeError."failing as requested".\n'
             'RuntimeError: failing as requested'
         )
-        self.assertEqual(len(calls), 1, f"expected 1 call, but got extra: {calls[1:]}")
+        self.assertEqual(
+            len(calls),
+            1,
+            f"expected 1 call, but got extra: {calls[1:]}")
```

Comment:
A couple of these formatting changes seem strictly worse than what we had before. Any reason for this change and the assertRegex one above?

Reply:
I'm just using the tooling (ie. `tox -e fmt`) not choosing any of this myself. Can we adjust the tools so that it matches what you like?

Reply:
I don't quite understand that, because we were already running `tox -e fmt` on all of this code, and there were no changes on this line and similar lines (`tox -e lint` checks that the formatting is as per `tox -e fmt` would make it).

When I check out this PR, change this line back to what it was, and run `tox -e fmt`, it doesn't reformat it.

Is it possible you modified this line so it was >100 columns, formatted, then modified it back? Or is something else going on?

Reply:
Hmm, that does do it. I don't recall, unfortunately :(. I worked on this module last week and then left it before picking it up this week, so it's not all totally fresh. Maybe I had a `type: ignore` and then removed that after in other modules we decided on the `assertEqual = unittest.TestCase.assertEqual` approach?

I'll clean up this PR and try to keep better track of this going forward.